src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -817,6 +823,16 @@ export class AppComponent implements OnInit, AfterViewChecked {
         });
     }
 
+    /**
+     * Resizes the results pane
+     */
+    resizeResults(): void {
+        const self = this;
```

Comment:
not sure why we need to add self in this case, since we're not using any lambdas or callbacks.  Also inconsistent self\this usage below.